CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# How to contribute?
+
+Thank you for taking the time to contribute to this project. 🎉
+
+In this repository, we use the [All Contributors Bot](https://allcontributors.org/) that help us recognise all contributors, even when they don't directly contribute on the repository.
+You can find all [emoji/Type keywords](https://allcontributors.org/docs/en/emoji-key) representing the types of contribution.
+To add a contributor, comment on Issue or Pull Request (where the contributor is involved) using this message for @all-contributors:
+`@all-contributors please add @<username> for <keyword in the Type column>`
+
+Whatever your availability, there is a way to contribute to the RSE TRE repo!
+
+## 👋 I'm busy, I only have 5 minutes
+
+Look through our currently open [issues](../../issues) to troubleshoot an issue or participate in an ongoing discussion by commenting.
+You can also share this repository with someone who might be interested in getting involved.
+
+## ⏳ I've got 15 minutes - tell me what I should do
+
+You can read our [README](./README.md) file to find details and the next milestones in the project.
```

Comment:
Should we remove the `I'm busy, I only have 5 minutes` and `I've got 15 minutes - tell me what I should do` sections? This makes sense for an active project with a good roadmap involving code/docs, but we're at a different stage. We're developing the initial community including deciding it's direction, so most of the "contributions" should be contributing to discussions and organising things, rather than contributin to the contents of the repo. Arguably contributing to Slack discussions is just as important.

Reply:
Have updated to say join discussion on Slack!